mono/metadata/icall.c

```diff
@@ -1293,6 +1293,27 @@ ves_icall_System_Runtime_CompilerServices_RuntimeHelpers_RunModuleConstructor (M
 	mono_runtime_class_init_full (vtable, error);
 }
 
+#ifdef ENABLE_NETCORE
+MonoBoolean
+ves_icall_System_Runtime_CompilerServices_RuntimeHelpers_SufficientExecutionStack (void)
```

Comment:
Functions that use the address of labels as stack pointer proxies should generally be marked as noinline.